classes/fakeroot.oeclass

```diff
@@ -3,7 +3,7 @@
 ## By inheriting this class, it is ensures that the native version of fakeroot
 ## build, is used instead of fakeroot from the build system.
 
-CLASS_DEPENDS += "native:fakeroot"
+CLASS_DEPENDS += "native:fakeroot native:libacl1-dev"
```

Comment:
we don't have a recipe for it, it is a build-host dep, and without it fakeroot messes up at least for debian like distroes.